lib/collection/src/collection_manager/optimizers/indexing_optimizer.rs

```diff
@@ -80,10 +85,14 @@ impl IndexingOptimizer {
                     StorageType::Mmap => true,
                 };
 
-                let big_for_mmap = vector_count >= self.thresholds_config.memmap_threshold;
-                let big_for_index = vector_count >= self.thresholds_config.indexing_threshold;
+                let big_for_mmap =
+                    vector_size >= self.thresholds_config.memmap_threshold * BYTES_IN_KB;
+                let big_for_index =
+                    vector_size >= self.thresholds_config.indexing_threshold * BYTES_IN_KB;
+
+                // ToDo: remove deprecated
```

Comment:
Difficult to understand what is deprecated, maybe link an issue?

Reply:
there are no issue, but it should be removed as a part of real time index integration